### PR TITLE
nextcloud-client: 3.2.2 -> 3.2.3

### PR DESCRIPTION
**Pull request by Kranzes**
###### Motivation for this change
Simple version update.
Inkscape dependency added due to nextcloud-client's new changes https://github.com/nextcloud/desktop/releases/tag/v3.2.3

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by caugner — APPROVED** [review]
LGTM

File: pkgs/applications/networking/nextcloud-client/default.nix [hunk 1]
```diff
@@ -53,6 +54,7 @@ mkDerivation rec {
     qtgraphicaleffects
     qtwebsockets
     sqlite
+    inkscape
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/nextcloud-client/default.nix:57]: If the icons are generated with inkscape this need to go to nativeBuildInputs.

Comment (Kranzes) [reply pkgs/applications/networking/nextcloud-client/default.nix:57]: Why is that, may you elaborate please? i dont quite understand all the differences between the available build inputs.

Comment (caugner) [reply pkgs/applications/networking/nextcloud-client/default.nix:57]: Good catch by @SuperSandro2000.

Inkscape should indeed only be required for building nextcloud-client, not for running the built nextcloud-client.

Comment (caugner) [reply pkgs/applications/networking/nextcloud-client/default.nix:57]: See: https://discourse.nixos.org/t/use-buildinputs-or-nativebuildinputs-for-nix-shell/8464

Comment (Kranzes) [reply pkgs/applications/networking/nextcloud-client/default.nix]: Oh i see, thanks a lot guys!

**Review by caugner — APPROVED** [review]

**Review by fufexan — APPROVED** [review]

**Pull request merged**